python/sparkdl/estimators/keras_image_file_estimator.py

```diff
@@ -285,7 +284,8 @@ def _name_value_map(paramMap):
         modelBytesBc = sc.broadcast(modelBytes)
 
         # Obtain params for this estimator instance
-        baseParams = _name_value_map(self.extractParamMap())
```

Comment:
nit/suggestion: maybe for consistency `_name_value_map` can filter out untunable params & we can continue to use `_name_value_map` here? thinking it'd be nice since we use `_name_value_map` above to process `paramMaps` and it would be good to handle both `paramMaps` and `self.extractParamMap()` consistently.

for clarity (since `_name_value_map` would now be doing more than just getting a name -> value map) we could rename `_name_value_map` -> `_get_tunable_name_value_map` or something

Reply:
@smurching, thanks for catching this!
@sueann, Yes! that's what we check in '_validateParams'.